packages/fuels/src/cli.ts

```diff
@@ -32,7 +32,6 @@ export const configureCli = () => {
 
   program.version(versions.FUELS, '-v, --version', 'Output the version number');
   program.helpOption('-h, --help', 'Display help');
-  program.addHelpCommand('help [command]', 'Display help for command');
```

Comment:
No need to remove this here, as this will be handled via:
 - https://github.com/FuelLabs/fuels-ts/pull/3190

Reply:
Cool, I've just changed it to use `helpCommand` instead of `addHelpCommand` for now

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3210/commits/3f10f1f88d6d6ef5a29ec891a801ad4598aa35ef